src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -22,11 +22,13 @@
 #include "parsing.h"
 #include "pgsetup.h"
 
+bool allowFailover = false;
```

Comment:
Nope. At the moment we have a not ideal situation where the command line option parsing is kept in sync with the configuration file parsing. That's a nice user friendly property, but what do we do about options that are not relevant to the configuration and only apply for the duration of an interactive command?

Well `allowFailover` falls into that bin, and as such we need to have a global variable for it. That's also because of the way we are handling command lines parsing function and `getopt` functions: those can't share state, so we need a global one.